### PR TITLE
Issues #9449, #8104: Prevent blocker stealing tab handling of the current focus root

### LINKED ISSUE #9449: Tab key not working if you click outside a modal window and then return to the window
in desktop:
when you open a window with controls in it, if you press tab key it will navigate correctly, but if you click outside the window and then click inside the window, the cursor remains on the textfield and will not move to the next control.
I notices also, not only the Tab key not working, also Space key not working when you press it on buttons.

### PR DESCRIPTION
**Pull request by level420**
If the widget attached to the blocker is also attached to the `qx.ui.core.FocusHandler` as a focus root, activating the blocker after this widget was activated, leads to the focus handler re-activate the widget behind the blocker, loosing tab handling for widget which is visually in front. Hence we prevent activating the blocker in this situation.

Fixes #9449, #8104

### CODE REVIEW

**Review by oetiker — APPROVED** [review]
sounds reasonable

**Review by cajus — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by derrell — APPROVED** [review]
I don't much care for big block comments in the middle of an expression / condition, so I'd prefer that the comment be moved just above the `if` statement. I don't want to block this for that reason, but if you don't object, please move the comment.

**Comment by level420** [comment]
@derrell will do this in a subsequent PR, as adding it now would restart the whole review request, discarding what was approved now.

**Comment by hkollmann** [comment]
Could we go the short way for merge or must we wait for @zaucker?
If not i can publish the sdk after merge

**Comment by cajus** [comment]
What about renaming the SDK to *qx-sdk*? Or do you want to use the current name as a test?

**Comment by hkollmann** [comment]
Let‘s start with qooxdoo-sdk. Otherwise we have to change Package.json again

**Comment by derrell** [comment]
@level420 Yeah, I thought about the whole starting-over thing. Subsequent PR sounds good.

@hkollmann We have enough approvals to merge, but it has not been the requisite time period. I intend to propose a change to the process to allow an immediate merge (no wait time) if *all* core team members have approved, but that is not currently in place, so the addition of zaucker's approval wouldn't speed things up at present.

**Review by zaucker — APPROVED** [review]

**Comment by hkollmann** [comment]
With new rule " Allow fast-track approvals" we can merge.
Thanks, @level420!

**Pull request merged**
